core/domain/takeout_service_test.py

```diff
@@ -987,14 +1113,16 @@ def test_export_data_for_full_user_nontrivial_is_correct(self):
                 'has_suggestion': False,
                 'summary': None,
                 'message_count': 2,
-                'last_updated_msec': utils.get_time_in_millisecs(
+                'last_updated': utils.get_time_in_millisecs(
```

Comment:
Why drop the msec? Historically it has been quite useful for distinguishing "datetime object" fields from millisec fields, so it would be nice to keep it.

Reply:
Done